### ISSUE TITLE
Master list enhancements

### ISSUE THREAD

**Issue by cdsouthan** [OpenSourceMalaria/OSM_To_Do_List#533]

**Comment by cdsouthan** [comment]
The order is my ranking but AWK  may not correspond to the real world fixing sequence.  I'm kicking off with outlines that can be expanded

1.  The biggest scientific  issue is whole parasite assay heterogeneity across the series.

**Comment by cdsouthan** [comment]
It has been brought to my attention (in a friendly way of course) that in order to get some engagement on the themes above I should split the objectives down to bite-size chunks.

**Comment by david1597** [comment]
Sure. Breaking this down will be the easiest way to go.

I'll start where you suggest and get the OSM numbers in the sheet.

**Comment by mattodd** [comment]
Agreed. To contain the task I'd recomment focussing on the Series 4 compounds for now. Also @cdsouthan just re the distinction between "final" compounds and synthetic intermediates. We have this already (fully - I think - for Series 4, less so for others) in that all biologically evaluated compounds have MMV numbers. Can we just use that as the filter?

**Comment by david1597** [comment]
OK, everything in the sheet now has its OSM number. There were a couple of Sydney compounds, around 20 Edinburgh compounds and around 100 inherited compounds. The inherited compounds were designated 'X', as we're not sure where they were synthesised.

**Comment by cdsouthan** [comment]
Good, While I woke up in the night wondering if adding a synonym was such a good idea after all (seeing as a general principle we want as few as humanly possible) - I hope  it helps in the end i.e.  that "we" have our full set of primary identifier ducks lined up.  OK, so mystery compounds seem somewhat paradoxical in open drug discovery (but lets hope the referees overlook that... ).  Moving swiftly on then;

2) Please do a referential integrity check that should include but is not restricted to  a) absence of gaps for the molecular specifications b) no entry or conversion corruptions c) duplicates in any columns d) the sheet behaves itself for common  operations e.g. text <> CSV < > Excel  <> Libre Office (or whatever) and that column sorting and setting tables also work OK.   Also archive a copy-just in case....  As ever @david1597 try to find an OSM friend to do this tedious but crucial x-checking with you ( @mattodd will buy them a beer and maybe a pizza also for you) 

n.b. Adding fresh synonyms raises the back-propagation issue.  At some time it wbgood to arrange that a named MMV person has a copy of our optimised sheet so they are on the record as being in possession of these new n2s mappings for their compounds

**Comment by david1597** [comment]
The integrity checks will likely come as we write up the experimental for the series 4 paper - @edwintse and myself should hopefully be getting through these in the near future.
